src/Eurofurence.App.Backoffice/Services/UserService.cs

```diff
@@ -13,5 +14,10 @@ public async Task<UserRecord> GetUserSelf()
             options.Converters.Add(new JsonStringEnumConverter());
             return await http.GetFromJsonAsync<UserRecord>("Users/:self", options) ?? new UserRecord();
         }
+        public async Task PutUserArtistAlleyStatusAsync(String userID, ArtistAlleyUserStatusRecord.UserStatus status)
```

Comment:
This does not seem to work correctly, as the controller is expecting a ArtistAlleyUserStatusChangeRequest, while the Backoffice service is only sending the UserStatus. So the wrapper class with the status + reason is missing.
The reason could be added to the confirm dialog as a text area.